content/design-systems-for-developers/react/ko/distribute.md

```diff
@@ -316,111 +318,119 @@ jobs:
+</div>
 
-## 앱에 디자인 시스템 임포트하기
+![디자인 시스템 import하기](/design-systems-for-developers/design-system-import.png)
+## 앱에 디자인 시스템 적용
 
-이제 디자인 시스템이 온라인에서 실행되고 있으니 디펜던시를 설치하고 UI 컴포넌트를 사용하는 것은 어렵지 않습니다.
+이제 디자인 시스템이 온라인에 있으니 의존성으로 설치하고, UI 컴포넌트로 사용하는 것은 간단합니다.
 
-#### 예제로 사용할 앱 준비하기
+#### 예제 앱 준비
 
-이 튜토리얼의 앞부분에서 우리는 프런트 엔드 기술로 인기가 많은 React와 styled-components를 기준으로 설정했습니다. 즉, 디자인 시스템의 장점을 최대한 많이 사용하기 위해서는 우리의 예제 앱도 React와 styled-components를 사용해야 합니다.
+이 튜토리얼의 앞 부분에서, 많이 쓰이는 프런트엔드 스택을 표준화를 했습니다. 이 스택에는 리액트(React)와 Styled Components가 포함되는데, 즉, 디자인 시스템을 최대한 활용하려면 예제 앱에서도 리액트 및 Styled Components를 사용해야 합니다.
 
-<div class="aside">Svelte나 다른 웹 컴포넌트들을 사용하면 프레임워크에 구애받지 않는 UI 컴포넌트를 만들 수도 있습니다. 하지만 그 방법들은 만들어진 지 얼마 되지 않았거나, 문서화가 부족하거나 다방면으로 적용되기 어려울 수 있기 때문에 이 가이드에는 아직 포함하지 않았습니다. </div>
+<div class="aside">
+💡 Svelte나 web components와 같이 다른 유망한 방법을 사용하면 프레임워크에 구애받지 않고 UI 컴포넌트를 구성할 수 있습니다. 그러나 Svelte나 web components는 상대적으로 새로운 방식이라 문서화가 덜 되었거나 널리 채택되지 않았기 때문에 이 가이드에는 아직 포함되지 않았습니다.
+</div>
 
-이 예제 앱은 [컴포넌트 주도 개발 Component-Driven Development](https://blog.hichroma.com/component-driven-development-ce1109d56c8e)를 할 수 있도록 스토리북을 사용합니다. 컴포넌트 주도 개발은 컴포넌트로 바닥부터 시작해서 페이지로 끝내는 UI 구축 방법입니다. 데모에서 우리는 두 개의 스토리북을 번갈아 가며 실행할 것입니다. 하나는 예제 앱, 하나는 디자인 시스템을 위한 것입니다.
+ 예제 앱은 스토리북을 사용하여 [컴포넌트 주도 개발 Component-Driven Development](https://www.componentdriven.org/)를 용이하게 합니다. 이 개발 방법은 UI를 아래에서부터, 즉 컴포넌트 개발부터 시작해서 페이지로 끝내는 방식입니다. 데모하는 동안에 두 개의 스토리북을 번갈아가며 실행합니다. 하나는 예제 앱을 위한 것이고 다른 하나는 디자인 시스템을 위한 것입니다. 
 
-예제 앱을 설정하기 위해 커맨드 라인에서 아래 명령어들을 실행합니다;.
+예재 앱을 설정하기 위해서 커맨드 라인에 다음과 같이 명령어를 실행합니다. - 
 
 ```shell
-// 로컬에 파일을 클론합니다.
+# Clones the files locally
 npx degit chromaui/learnstorybook-design-system-example-app example-app
-cd example-app
-# 디펜던시를 설치합니다
-yarn install
-## 스토리북을 시작합니다
-yarn storybook
-```
 
-디펜던시를 설치하고 앱에서 스토리북을 시작합니다.
+cd example-app
 
-```shell
+# Install the dependencies
 yarn install
+
+## Start Storybook
 yarn storybook
 ```
 
-앱이 사용하는 간단한 컴포넌트들의 스토리가 실행되는 게 보일 것입니다.
+앱에서 사용되는 단순한 컴포넌트의 스토리들(stories)이 스토리북에서 실행되는 것이 보일 것입니다. - 
 
-![Initial storybook for example app](/design-systems-for-developers/example-app-starting-storybook-6-0.png)
+![예제 앱을 위한 초기 스토리북](/design-systems-for-developers/example-app-starting-storybook-6-0.png)
 
-<h4>디자인 시스템 통합하기</h4>
+<h4>디자인 시스템 통합</h4>
 
-디자인 시스템의 스토리북이 배포되었으니 예제 앱에 추가하도록 합니다. 예제 앱의 `.storybook/main.js` 파일을 아래와 같이 업데이트하면 스토리북을 추가할 수 있습니다.
+작업한 디자인 시스템의 스토리북이 배포되어 있을 것입니다. 이제 스토리북을 예제 앱에 추가해보세요. 예제 앱의 `.storybook/main.js`를 다음과 같이 업데이트할 수 있습니다. - 
 
-```javascript
+```diff:title=.storybook/main.js
 // .storybook/main.js
 
 module.exports = {
   stories: ['../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  refs: {
-    'design-system': {
-      title: 'My design system',
-      // 배포되었을 때 Chromatic에서 제공한 url
-      url: 'https://your-published-url.chromatic.com',
-    },
-  },
++ refs: {
++   'design-system': {
++     title: 'My design system',
++     //👇 Chromatic으로 배포된 url
++     url: 'https://your-published-url.chromatic.com',
++   },
++ },
```

Comment:
@lisohh could you revert the comment back to English if you wouldn't mind?

Reply:
thanks for your feedback. I was the one who reverted the Korean comments to English ones and it seems that I've missed some parts. I'll fix those ASAP.